### PR TITLE
Update dependencies, bring .travis config into 2015

### PR DESCRIPTION
**Pull request by cliftonc**
Unclear if this project has been abandoned (be happy to help out), but all of the dependencies and core configuration are a very long way out of date.

This request updates the three core dependencies, and updates Travis to work on current versions of Node that most people are likely to be running in production.

### CODE REVIEW

**Comment by andreas-marschke** [comment]
+1

**Comment by rfk** [comment]
Thanks!  It's certainly not abandoned, we're using it in production, although I suspect it's mostly in "things just work so it doesn't get much attention" mode these days.

@zaach are you the closest we have to a "module owner" on this project today?

**Comment by cliftonc** [comment]
The main reason (selfishly) I want this to be accepted, is that the old version of the plist dependency contains a 26mb test file that is making a lot of our services much bigger than they need to be, and with a very distributed team on sometimes not great connections this does surprisingly make a difference.

We're starting to build a lot around this and so was also keen to give it a nudge in comparison to see if there are signs of life.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: node_js
 
 node_js:
- - 0.6
- - 0.8
+ - "1.6"
```
Comment (rfk) [inline .travis.yml]: I can't find it in the travis docs, what does "1.6" build here?

**Comment by rfk** [comment]
Otherwise this LGTM and I don't see much risk to merging

File: .travis.yml (hunk 1 above)
Comment (pdehaan) [inline .travis.yml]: @rfk: I reckon that'd be an io.js version number.

**Comment by rfk** [comment]
anyway, travis says it's green so imma merge this thing; thanks again @cliftonc

**Comment by cliftonc** [comment]
Fantastic - thanks!

**Comment by zaach** [comment]
New version published. :+1:

**Pull request merged**